sklearn/datasets/openml.py

```diff
@@ -489,24 +590,41 @@ def fetch_openml(name=None, version='active', data_id=None, data_home=None,
         If True, returns ``(data, target)`` instead of a Bunch object. See
         below for more information about the `data` and `target` objects.
 
+    return_frame : boolean, default=False
+        If True, returns a Bunch where the data attribute is a pandas
+        DataFrame.
+
+    nrows : int, default=5000
```

Comment:
Maybe rather `chunksize` as how you called it above? 
For an `nrows` argument, I would expect that it specifies to *only* return me that many (first) rows.